xml/System.Security.Cryptography/SafeEvpPKeyHandle.xml

```diff
@@ -260,11 +260,21 @@
         <Parameter Name="keyUri" Type="System.String" Index="1" FrameworkAlternate="net-9.0" />
       </Parameters>
       <Docs>
-        <param name="providerName">To be added.</param>
-        <param name="keyUri">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="providerName">The name of the <code>OSSL_PROVIDER</code> to process the key open request.</param>
+        <param name="keyUri">The URI assigned by the <code>OSSL_PROVIDER</code> of the key to open.</param>
+        <summary>Open a named key using a named <code>OSSL_PROVIDER</code>.</summary>
```

Comment:
Originally, this said "Open a named _public_ key..." however it is my understanding that `OpenKeyFromProvider` does not care if it is public or private. I assumed this to be a mistake when copying from the ENGINE documentation.